src/System/TimedScopes/TimedScopeProvider.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Omex.System.Diagnostics;
+using Microsoft.Omex.System.Logging;
+using Microsoft.Omex.System.TimedScopes.ReplayEventLogging;
+using Microsoft.Omex.System.Validation;
+
+namespace Microsoft.Omex.System.TimedScopes
+{
+	/// <summary>
+	/// Correlation Data
+	/// </summary>
+	public class TimedScopeProvider : ITimedScopeProvider
+	{
+		/// <summary>
+		/// Create a timed scope
+		/// </summary>
+		/// <param name="customLogger">Use a custom logger for the timed scope</param>
```

Comment:
>customLogger [](start = 19, length = 12)

The parameters should be in the same order in the comments and in the constructor

Reply:
Ordered, thanks you